### ISSUE TITLE
Threshold setting

### ISSUE THREAD

**Issue by chenhbo** [mabaorui/NeuralPull#17]
Hi, thank you for this nice work.
And thank you for sharing this code.

I also have a concern about the threshold mentioned in this issue. #8 

As you discussed before, this problem is because of the loss function of Eq.(2).

I visualized the learned sdfs, and found that almost all were positive, no negtive.
If this is still in a sdf field not udf, it means that NP prefers to learn positive sdf?

Thank you very much.

Looking forward to your reply!

**Comment by chenhbo** [comment]
By the way, I use the PyTorch version.
